lib/ExpandableListItem/ExpandableListItem.js

```diff
@@ -289,6 +290,19 @@ module.exports = kind(
 			this.bubble('onRequestScrollIntoView');
 		}
 	},
+	
+	/**
+	* Check for the first item in the client area, and close the list
+	* if the focus moves out of the drawer and auto-collapse is true
+	*
+	* @private
+	*/
+	spotlightUp: function(inSender, inEvent) {
+		var children = Spotlight.getChildren(this.$.client);
+		if (children.length && inEvent.originator == children[0] && this.getAutoCollapse() && this.getOpen()) {
```

Comment:
@rajyavardhanp I think below condition sequence looks better because it doesn't need to check first item if autoCollapse is false
{code}
if (this.getAutoCollapse() && this.getOpen() && children.length && inEvent.originator == children[0])
{code}